attributes/plugins.rb

```diff
@@ -5,13 +5,15 @@
 #     println ("${plugin.getShortName()}:${plugin.getVersion()}")
 # }
 default['osl-jenkins']['restart_plugins'] = %w(
-  credentials:2.1.13
+  structs:1.14
+  credentials:2.1.16
```

Comment:
It looks like version [2.1.15](https://plugins.jenkins.io/credentials) might end up breaking things, will this need to be handled?

Reply:
I'm not sure I'm following what you're referring to here.

Reply:
This is specifically what I'm referencing:
```
NOTE: all existing credentials fingerprint tracking history will be lost.
```

Reply:
I don't think we need to worry about that.